hack/gcp-secret-manager/commands/list.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+import click
+import requests
+import yaml
+from google.api_core.exceptions import PermissionDenied
+from google.cloud import secretmanager
+from util import CONFIG_PATH, PROJECT_ID, ensure_authentication, validate_collection
+
+
+@click.command("list")
+@click.option(
+    "-o",
+    "--output",
+    type=click.Choice(["json", "text"], case_sensitive=False),
+    default="text",
+    help="Output format, defaults to plain text but can be set to 'json'. Only applicable when a collection is specified.",
+)
+@click.option("-c", "--collection", default="", help="Name of the secret collection")
```

Comment:
As per design document, it seems we have to consider also the `--group=foobar` or `-g foobar` flag. As an example
```sh
$ hack/secret-manager.sh list --group test-platform-ci-sudoers
test-platform-ci-sudoers:
- foo
- bar
...
```